reposilite-plugins/javadoc-plugin/src/main/kotlin/com/reposilite/plugin/javadoc/DocExtractor.kt

```diff
@@ -0,0 +1,51 @@
+package com.reposilite.plugin.javadoc
+
+import java.io.File
+import java.io.IOException
+import java.nio.file.Path
+import java.util.jar.JarFile
+
+/**
+ * @author TOTHTOMI
+ * @version 1.0.0
+ * @since Dec. 27, 2021
+ */
```

Comment:
We don't really use authority comments, it's hard to maintain and doesn't provide valid data for long living projects - authors are listed in git history per each file :)

Reply:
Ok, that's autogenerated by my Intellij, but I will remove it.